dask_cuda/cli/dask_cuda_worker.py

```diff
@@ -19,154 +19,160 @@
+    when the worker starts spilling to host memory; can be an integer (bytes), float
+    (fraction of total device memory), string (like ``5GB`` or ``5000M``), ``auto``,
+    or 0 to disable spilling to host (i.e., allow full device memory usage)""",
 )
 @click.option(
     "--rmm-pool-size",
     default=None,
-    help="If specified, initialize each worker with an RMM pool of "
-    "the given size, otherwise no RMM pool is created. This can be "
-    "an integer (bytes) or string (like 5GB or 5000M)."
-    "NOTE: This size is a per worker (i.e., per GPU) configuration, "
-    "and not cluster-wide!",
+    help="""If specified, initialize each worker with an RMM pool of the given size; can
+    be an integer (bytes) or string (like ``5GB`` or ``5000M``); this size is per-worker
+    (GPU) and not cluster-wide
+
+    .. note::
+        This size is a per-worker (GPU) configuration, and not cluster-wide""",
 )
 @click.option(
     "--rmm-managed-memory/--no-rmm-managed-memory",
     default=False,
-    help="If enabled, initialize each worker with RMM and set it to "
-    "use managed memory. If disabled, RMM may still be used if "
-    "--rmm-pool-size is specified, but in that case with default "
-    "(non-managed) memory type."
-    "WARNING: managed memory is currently incompatible with NVLink, "
-    "trying to enable both will result in an exception.",
+    show_default=True,
+    help="""If enabled, initialize each worker with RMM and set it to use managed
+    memory; if disabled, RMM may still be used by specifying ``--rmm-pool-size``
+
+    .. warning::
+        Managed memory is currently incompatible with NVLink; trying to enable both will
+        result in an exception.""",
 )
 @click.option(
     "--rmm-log-directory",
     default=None,
-    help="Directory to write per-worker RMM log files to; the client "
-    "and scheduler are not logged here."
-    "NOTE: Logging will only be enabled if --rmm-pool-size or "
-    "--rmm-managed-memory are specified.",
+    help="""Directory to write per-worker RMM log files to; the client
+    and scheduler are not logged here.
+
+    .. note::
+        Logging will only be enabled if ``--rmm-pool-size`` or ``--rmm-managed-memory``
+        are specified.""",
+)
+@click.option("--pid-file", type=str, default="", help="File to write the process PID")
+@click.option(
+    "--resources",
+    type=str,
+    default="",
+    help="""Resources for task constraints like ``GPU=2 MEM=10e9``; resources are
+    applied separately to each worker process (only relevant when starting multiple
+    worker processes with ``--nprocs``)""",
 )
 @click.option(
-    "--reconnect/--no-reconnect",
+    "--dashboard/--no-dashboard",
+    "dashboard",
     default=True,
-    help="Reconnect to scheduler if disconnected",
+    show_default=True,
+    required=False,
+    help="Launch the dashboard",
+)
+@click.option(
+    "--dashboard-address",
+    type=str,
+    default=":0",
```

Comment:
Does ":0" mean a random port will be assigned? If so, is that what we want or do we want to keep Distributed's default (":8787")? Also, are you sure the colon is required/supported here?

Reply:
Not sure, this was already set when I edited the file - looking at the blame, this was done in #70 for a hard-coded example, so maybe it's good to change this to match with Distributed's default now.

Reply:
Hmm, ok, I thought that was being added just now, I initially missed the removal lines, so let's keep it as is for now.

Reply:
Would this be good to open a separate issue about? Not sure if this results in weird default behavior for the dashboard.

Reply:
Actually, I see that this is also the default for Distributed, so I'm going to assume this is a safe default.